source/blog/2020-06-24-release-notes.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: June 2020 Release Notes - Version 20.6.0
+published: true
+post_author:
+  display_name: Ryan O'Neill
+  twitter: ryanoneill
+tags: Releases, Finagle, Finatra, Scrooge, TwitterServer, Util
+---
+
+Summer is happening 🏖️, and with that comes the June release of Finagle, Finatra, Scrooge, Twitter Server, and Uti 😎.
```

Comment:
typo at the end. `Util`

Reply:
oh weird, it must have pasted over when I added the emoji.

Reply:
Thanks Hamdi.